builtin/logical/pki/path_acme_eab.go

```diff
@@ -36,16 +33,35 @@ func pathAcmeEabCreateList(b *backend) *framework.Path {
 		Operations: map[logical.Operation]framework.OperationHandler{
 			logical.ListOperation: &framework.PathOperation{
 				DisplayAttrs: &framework.DisplayAttributes{
-					OperationSuffix: "acme-configuration",
+					OperationVerb:   "list-eab-key",
+					OperationSuffix: "acme",
 				},
 				Callback: b.pathAcmeListEab,
 			},
+		},
+
+		HelpSynopsis:    "list external account bindings to be used for ACME",
+		HelpDescription: `list identifiers that have been generated but yet to be used.`,
+	}
+}
+
+func pathAcmeEabCreate(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "acme/eab",
```

Comment:
Wanna make this `new-eab` just to separate it out? I think `* acme/eab*` will be an effective policy for admins, but we'd want to avoid an accidental `LIST acme/eab` for users perhaps? 

Users won't really need to list EABs, IMO. My 2c. but let me know your thoughts :-)

Reply:
Yup I agree, implemented in https://github.com/hashicorp/vault/pull/20755/commits/6248cda1233b35c4d1684cd510cfc8c291919099